docs/Contributing/product-groups/software/README.md

```diff
@@ -0,0 +1,15 @@
+# Software
+
+This directory contains documentation specific to Fleet's Software product group.
+
+## Overview
+
+Fleet's Software functionality allows organizations to manage software across their device fleet, including software inventory, vulnerability management, and software installation.
+
+## Contents
+
+- [Software Vulnerability Processing](vulnerability-management-development.md)
```

Comment:
May want to move this into architecture in place of the existing article (and update links as appropriate).

This means that we don't have any articles in our dir yet...which sounds accurate, given that most of the documentation we'd want would likely be in the architecture section (with the exception of Fleet-maintained app manifest docs, which are in `ee/maintained-apps`.

Reply:
Yeah, I think you're right. Moving.